src/router/index.js

```diff
@@ -10,6 +11,11 @@ const routes = [
     name: 'home',
     component: HomeView
   },
+  {
+    path: '/movie/:id',
+    name: 'movie-page',
+    component: MoviePageView,
```

Comment:
It's not required for now, but better to use the `import()` functions for routes. As example you can see how it made for `/about` route